docs/source/operators/deploy-kubernetes.md

```diff
@@ -722,12 +722,12 @@ metadata:
 spec:
   restartPolicy: Never
   serviceAccountName: "{{ kernel_service_account_name }}"
-  {% if kernel_uid is defined or kernel_gid is defined %}
+  {% if kernel_uid or kernel_gid %}
   securityContext:
-    {% if kernel_uid is defined %}
+    {% if kernel_uid %}
```

Comment:
I think `kernel_uid` and `kernel_gid` should be settable to 0 as well -although it's never recommended to run as `root`, we shouldn't preclude it.

Reply:
Oh interesting, didn't know that, fixed